mono/metadata/class.c

```diff
@@ -2020,10 +1941,10 @@ mono_array_class_get (MonoClass *eclass, guint32 rank)
 gint32
 mono_class_instance_size (MonoClass *klass)
 {	
-	if (!klass->size_inited)
+	if (!m_class_is_size_inited (klass))
```

Comment:
Should we extract that into a `mono_class_setup_size`, and have it be the following?
```
void
mono_class_setup_size (MonoClass *klass)
{
    if (m_class_is_size_inited (klass))
        return;

    mono_class_init (klass);
}
```